frontend_example_keycloak_integration/ersms-fe/src/keycloak.tsx

```diff
@@ -0,0 +1,13 @@
+import Keycloak from 'keycloak-js';
+
+// Setup Keycloak instance as needed
+// Pass initialization options as required or leave blank to load from 'keycloak.json'
+
+// TODO: configurable keycloak url - maybe use 'keycloak.json'
```

Comment:
I agree
If we want to use this in our main frontend we should be able to configure the address with a single config file
We could add support for different profiles like local / docker / k8s